gcs/pom.xml

```diff
@@ -220,9 +223,12 @@
                   <include>com.google.oauth-client</include>
                   <include>com.google.protobuf</include>
                   <include>com.google.storage.v2</include>
+                  <include>com.lmax</include>
                   <include>io.grpc</include>
                   <include>io.opencensus</include>
                   <include>io.perfmark</include>
+                  <include>org.apache.httpcomponents</include>
```

Comment:
Why do we need Apache HTTP client?

Reply:
I can not find where this dependency declared. May you share dependency chain that brings this in?

Reply:
Cloudmonitoring exporter relies on [cloud-core](https://mvnrepository.com/artifact/com.google.cloud/google-cloud-core/2.5.4) which relies on [http-client](https://mvnrepository.com/artifact/com.google.http-client/google-http-client/1.41.4) that has the dependency. I noticed runtime errors because of which i had to add this inclusion

Reply:
It isn't a direct dependency. [Google-http-client](https://mvnrepository.com/artifact/com.google.http-client/google-http-client/1.41.4) depends on it